appveyor.yml

```diff
@@ -0,0 +1,65 @@
+environment:
+
+  global:
+    # SDK v7.0 MSVC Express 2008's SetEnv.cmd script will fail if the
+    # /E:ON and /V:ON options are not enabled in the batch script intepreter
+    # See: http://stackoverflow.com/a/13751649/163740
+    CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\appveyor\\run_with_env.cmd"
+
+  matrix:
+    - PYTHON: "C:\\Miniconda-x64"
+      PYTHON_VERSION: "2.7"
+      PYTHON_ARCH: "64"
+      CONDA_PY: "27"
+
+    - PYTHON: "C:\\Miniconda35-x64"
+      PYTHON_VERSION: "3.5"
+      PYTHON_ARCH: "64"
+      CONDA_PY: "35"
+
+    - PYTHON: "C:\\Miniconda36-x64"
+      PYTHON_VERSION: "3.6"
+      PYTHON_ARCH: "64"
+      CONDA_PY: "36"
+
+    - PYTHON: "C:\\Miniconda36-x64"
+      PYTHON_VERSION: "3.7"
+      PYTHON_ARCH: "64"
+      CONDA_PY: "37"
+
+install:
+  - ECHO "Filesystem root:"
+  - ps: "ls \"C:/\""
+
+  - ECHO "Installed SDKs:"
+  - ps: "ls \"C:/Program Files/Microsoft SDKs/Windows\""
+
+  # This upates conda and installs the necessary packages.
+  - "%PYTHON%\\Scripts\\conda.exe update --yes conda"
+  - "%PYTHON%\\Scripts\\conda.exe config --prepend channels conda-forge"
+  - '%PYTHON%\\Scripts\\conda.exe install --yes numpy cython future six setuptools sphinx numpydoc'
+  - "%PYTHON%\\Scripts\\conda.exe update -y --all"
+  - "%PYTHON%\\Scripts\\conda.exe info"
+  - "%PYTHON%\\Scripts\\conda.exe list"
+  - ps: Start-FileDownload 'https://www.coin-or.org/download/binary/Ipopt/Ipopt-3.11.0-Win32-Win64-dll.7z'
+  - 7z x Ipopt-3.11.0-Win32-Win64-dll.7z
```

Comment:
I need to figure out how to unzip this such that the `include` and `lib` directories are beside the `setup.py` file. Another approach would be to introduce an environment variable that a use could set which points to the ipopt directory.